equipment-service/src/main/resources/equipment-service-openapi.yaml

```diff
@@ -899,66 +899,21 @@ paths:
       - name: equipmentType
         in: query
         description: equipment type
-        required: true
+        required: false
```

Comment:
This uncovered an older bug in RDBMS implementation (which was not triggered before because equipmentType was a required parameter) - now the DAO throws NPE if equipmentType is not provided. Please fix that bug in the DAO (I know, you did not create it in the first place).